### PR TITLE
Fixes #37997 - Support Rails 7.0

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

**Review by ekohl — APPROVED** [review]
This feels compatible with Rails 6.1, so let's get it merged without it?

**Comment by ofedoren** [comment]
Removed the commit for testing with opened PR, ready for merge.

File: test/integration/discovered_hosts_test.rb [hunk 1]
```diff
@@ -42,7 +42,9 @@ class DiscoveredHostsTest < IntegrationTestWithJavascript
       page.find_link('Reboot').click
       wait_for_ajax
       assert page.has_text?('The following hosts are about to be changed')
-      page.find_button('Submit').click
+      wait_for do
```
Comment (stejskalleos) [inline test/integration/discovered_hosts_test.rb:45]: What's the reason to put this particular click action into the `wait_for` block?
Other tests have it as well; should we update them as well?

Comment (ofedoren) [reply test/integration/discovered_hosts_test.rb:45]: For some reason some integration tests fail either on timeouts or (like in this case) the app seems to not respond 
(with either data or page) in a certain time. Not sure if it's frontend or backend related, but waiting for a bit more helps :/

Comment (stejskalleos) [reply test/integration/discovered_hosts_test.rb:45]: Oh yes, the daily joys of maintaining integration tests.
If it helps, +1 from me.

**Review by stejskalleos — APPROVED** [review]

**Pull request merged**